lib/cmsis_rtos_v1/cmsis_thread.c

```diff
@@ -101,7 +101,7 @@ osPriority osThreadGetPriority(osThreadId thread_id)
 	k_tid_t thread = (k_tid_t)thread_id;
 	u32_t priority;
 
-	if (_is_in_isr()) {
+	if ((!thread_id) || (_is_in_isr())) {
```

Comment:
Use thread_id == NULL to preserve uniformity

Reply:
Done